allure-app-under-test/src/main/java/com/capgemini/ntc/selenium/pages/projectY/ChallengingDomPage.java

```diff
@@ -0,0 +1,58 @@
+package com.capgemini.ntc.selenium.pages.projectY;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import com.capgemini.ntc.selenium.core.BasePage;
+import com.capgemini.ntc.selenium.pages.environment.GetEnvironmentParam;
+import com.capgemini.ntc.selenium.pages.environment.PageSubURLsProjectYEnum;
+import com.capgemini.ntc.test.core.logger.BFLogger;
+
+public class ChallengingDomPage extends BasePage {
+	
+	private final By	selectorAllCellsInRows	= By.cssSelector("tbody > tr");
+	private By			selectorFirstButton		= By.cssSelector(".large-2.columns > .button:nth-child(1)");
+	
+	@Override
+	public boolean isLoaded() {
+		BFLogger.logDebug("The ChallengingDOM page is loaded.");
+		return getDriver().getCurrentUrl()
+						.equals(GetEnvironmentParam.THE_INTERNET_MAIN_PAGE.getValue() + PageSubURLsProjectYEnum.CHALLENGING_DOM.getValue());
+	}
+	
+	@Override
+	public void load() {
+		BFLogger.logDebug("load()");
```

Comment:
You can add something like 
getDriver().get(GetEnvironmentParam.THE_INTERNET_MAIN_PAGE.getValue() + PageSubURLsProjectYEnum.CHALLENGING_DOM.getValue());